ingestion/src/metadata/ingestion/source/database/processor.py

```diff
@@ -34,6 +36,12 @@
 PII = "PII"
 
 
+class Scanner(ABC):
```

Comment:
why do we need an abstract class if we're only defining a scanner? Are we planning to have more going forward?

Reply:
No , yes will remove `scanner` class